backend/tests/clubs/test_commands.py

```diff
@@ -651,3 +651,46 @@ def test_wrong_arguments(self):
 
         with self.assertRaises(CommandError):
             call_command("merge_duplicates", "--tag")
+
+
+class ExpireMembershipInvitesTest(TestCase):
+    def setUp(self):
+        # Users
+        self.user1 = get_user_model().objects.create_user(
+            "bfranklin", "bfranklin@seas.upenn.edu", "test"
+        )
+        self.user2 = get_user_model().objects.create_user(
+            "tjefferson", "test-application-notif@example.com", "test"
+        )
+
+        # Club
+        self.club1 = Club.objects.create(
+            code="one", name="Club One", active=True, email="test@example.com"
+        )
+
+        # Expired MembershipInvite
+        self.expired_invite = MembershipInvite.objects.create(
+            email=self.user2.email,
+            club=self.club1,
+            creator=self.user1,
+            active=True,
+            expires_at=timezone.now() - datetime.timedelta(days=1),
+        )
+
+        # Active MembershipInvite
+        self.active_invite = MembershipInvite.objects.create(
+            email=self.user1.email,
+            club=self.club1,
+            creator=self.user1,
+            active=True,
+            expires_at=timezone.now() + datetime.timedelta(days=1),
+        )
+
+    def test_expire_membership_invites(self):
```

Comment:
nit: would be great to have a comment here explaining what expected behavior of the command is. It's self-explanatory for this test (and we haven't done a great job of this in the past), but it'd be nice to set the standard going forwards.